api/package.json

```diff
@@ -16,9 +16,11 @@
     "lambda-api": "^1.0.1"
   },
   "devDependencies": {
+    "@swc/cli": "^0.1.62",
+    "@swc/core": "^1.3.62",
     "@types/aws-lambda": "^8.10.114",
     "@types/node": "^18.16.2",
     "fastify": "^4.17.0",
-    "ts-node": "^10.9.1"
+    "tsx": "^3.12.7"
```

Comment:
These are newer packages that I wanna try out as part of our side project. Just for learning purposes

Reply:
Agreed